tests/spiegelman_material.cc

```diff
@@ -0,0 +1,289 @@
+#include <aspect/simulator.h>
```

Comment:
Is this a copy of the `benchmarks/spiegelman_et_al_2016_gcubed/spiegelman_material.cc`? If so, replace the whole file by one with
`#include <../benchmarks/spiegelman_et_al_2016_gcubed/spiegelman_material.cc` to avoid the two files getting out of sync.

Reply:
no, it just has the same name. This one actually tests the derivatives of `../benchmarks/spiegelman_et_al_2016_gcubed/spiegelman_material.cc` :)